docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -87,13 +94,25 @@ TIP: To learn more about the query, select the **Threat Intel** tab, then click
 - `url.full`: The full URL of the event source
 - `registry.path`: The full registry path, including the hive, key, and value
 
-Additional threat intelligence data is generated for indicator match alerts when indicators are matched with source events, as outlined by the conditions in the relevant indicator match rule.
+If these fields aren't present, {es-sec} does not perform the query and threat intelligence for the alert is not displayed.
+
+TIP: To learn more about the query, select the **Threat Intel** tab, then click the **Inspect** button next to the threat you want to inspect.
+
+Additional threat intelligence data is generated for indicator match alerts when indicators are matched with source events, as outlined by the conditions in the relevant indicator match rule. Indicator match alerts always populate the Threat Matched Detected section under the Threat Threat Intel tab with additional threat match data.
 
 *Threat Intel*
 
-Shows the number of individual threats matching the alert in reverse chronological order, with the most recent threats at the top. Each threat is labelled with values from the `matched.field` and `matched.atomic` fields. All mapped fields are displayed for each matched threat.
+Shows the number of individual threats matching the alert. You can expand and collapse threat details by clicking the arrow icon at the end of the threat label. Each threat is labelled with values from the `matched.field` and `matched.atomic` fields and displays the threat intelligence provider. If the alert does not contain threat intelligence data, the *Threat Intel* tab displays a message that none is available.
+
+Matched threats are organized into several sections, described below. Within each section, matched threats are shown in reverse chronological order, with the most recent at the top. All mapped fields are displayed for each matched threat.
+
+* *Threat Matched Detected*: This section is solely reserved for threat indicator matches identified by an indicator match rule. Threat indicator matches are produced whenever event data matches a threat indicator field value in your indicator index. If indicator threat matches are not discovered, the section displays a message that none are available.
+* *Enriched with Threat Intelligence*: This section shows indicator matches that {es-sec} found when querying the alert for fields with threat intelligence. You can use the date time picker to modify the query timeframe, which looks at the past 30 days by default. Click the **Inspect** button, located on the far right of the threat label, to view more information on the query. If threat matches are not discovered within the selected timeframe, the section displays a message that none are available.
+* *Other*: This section shows threats that do not match any threat indicators or threat intelligence fields that {es-sec} queries the alert for. If these types of threat matches are not discovered on the alert, the section does not display.
```

Comment:
`This section shows threats that do not match any threat indicators` 
hmm I am not sure how accurate that statement is. Frankly I think it's OK to leave out the "other" section as it is primarily placed in the code base not as a product requirement but more of handling an edge case of finding threats on an alert that we as Elastic haven't enriched. So if an alert just randomly happens to have some threat enrichments on it, we don't discard them, but it is quite unlikely for enrichments to be present when we aren't adding them, if that makes sense.

Reply:
Makes sense, and I can remove this area entirely if we don't want to publicize it. If it's important to keep, I can reduce the text and insert it as a note after the list.

Reply:
You're correct! Removing it now, thanks. :)

Reply:
👍 to deleting it